control/control.CASP.xml

```diff
@@ -135,12 +135,12 @@ textdomain="control"
         <try_separate_home config:type="boolean">false</try_separate_home>
         <limit_try_home>5GB</limit_try_home>
         <root_space_percent>40</root_space_percent>
-        <root_base_size>3GB</root_base_size>
+        <root_base_size>10GB</root_base_size>
         <root_max_size>10GB</root_max_size>
         <proposal_lvm config:type="boolean">false</proposal_lvm>
         <vm_desired_size>15GB</vm_desired_size>
         <vm_home_max_size>25GB</vm_home_max_size>
-        <btrfs_increase_percentage config:type="integer">300</btrfs_increase_percentage>
+        <btrfs_increase_percentage config:type="integer">0</btrfs_increase_percentage>
```

Comment:
np: I didn't found documentation about this element in [yast2-installation documentation](https://github.com/yast/yast-installation/blob/SLE-12-SP2-CASP/doc/control-file.md).